### PR TITLE
add platform flag to docker commands to  enable M1 machines

### PR DESCRIPTION
**Pull request by annalezhikova**
If no flag provided the default platform is used.
For M1 machines it's arm64/v8, not amd64 like for most other devices.
This change will allow to run Makefile on M1 machine without affecting everyone else.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,9 +34,9 @@ clean:  ## Clean build artifacts
 
 .PHONY: archive
 archive: clean  ## Create the archive for AWS lambda
-	docker build -t bucket-antivirus-function:latest .
+	docker build --platform linux/amd64 -t bucket-antivirus-function:latest .
```
Comment (rodgerhe) [inline Makefile:37]: Does this impact non-M1 chip MacBooks?

**Comment by rodgerhe** [comment]
We maintain an internal branch `reejig-master` for our version. And use `master` to keep in sync with the upstream.

File: Makefile (hunk 1 above)
Comment (rodgerhe) [reply Makefile:37]: Just saw the comment, all good.

Comment (annalezhikova) [reply Makefile:37]: should not. Assumably the default for non M1s is linux/amd64, so it's overriding the default with the same value

**Comment by annalezhikova** [comment]
> We maintain an internal branch `reejig-master` for our version. And use `master` to keep in sync with the upstream.

Updated

**Review by rodgerhe — APPROVED** [review]

**Pull request merged**